docs/openapi-docs/src/test/scala/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -274,8 +274,6 @@ case class Person(name: String, age: Int) extends Entity
 case class Organization(name: String) extends Entity
 
 sealed trait ErrorInfo
-object ErrorInfo {
-  case class NotFound(what: String) extends ErrorInfo
-  case class Unauthorized(realm: String) extends ErrorInfo
-  case class Unknown(code: Int, msg: String) extends ErrorInfo
-}
+case class NotFound(what: String) extends ErrorInfo
```

Comment:
anything wrong with nested classes and 2.11?

Reply:
Sometimes, this is related to the first issue I mention in description.